module/users/helper.js

```diff
@@ -467,6 +467,65 @@ module.exports = class UsersHelper {
           });
         }
 
+        let totalCount = 0;
+        let mergedData = [];
+        //fetch all targeted programs to users
+        let targetedPrograms = await programsHelper.forUserRoleAndLocation(
+          requestedData,
+          "", // not passing page size
+          "", // not passing page number
+          "",
+          ["_id"],
+          programId
+        );
+        let targetedProgramIds = gen.utils.convertArrayObjectIdtoStringOfObjectId(targetedPrograms.data);
```

Comment:
@ankitpws if we don't do this it will anyways be converted to string while calling other service right?

Reply:
actually its not needed as we have extended forUserRoleAndLocation function to take program id.